app/jobs/geocode_firm_job.rb

```diff
@@ -0,0 +1,19 @@
+class GeocodeFirmJob < ActiveJob::Base
+  def perform(firm)
+    results = Geocoder.search(firm.full_street_address)
+
+    if results.any?
+      Stats.increment('radsignup.geocode.firm.success')
```

Comment:
Maybe we can push this into a helper here and use `stat :success` or `stat :failure` and push the duplication of keys down?